helm/ffc-pay-demographics/values.yaml

```diff
@@ -26,6 +26,7 @@ container:
   limitMemory: 100Mi
   limitCpu: 100m
   enabled: true
+  azureStorageAccountName: ffcpay
```

Comment:
could you please confirm that actual account name is not exposed.

Reply:
hey Abi, no, this will get overwritten by the platform code upon deployment